zookeeper-contrib/zookeeper-contrib-zooinspector/pom.xml

```diff
@@ -91,13 +91,9 @@
     </dependency>
     <dependency>
       <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-core</artifactId>
-      <exclusions>
-        <exclusion>
-          <groupId>*</groupId>
-          <artifactId>*</artifactId>
-        </exclusion>
-      </exclusions>
+      <artifactId>logback-classic</artifactId>
+      <scope>runtime</scope>
```

Comment:
This is marked as optional runtime in ZOOKEEPER-4820(#2155). Seems to fix test failure, but I have no cue to this. See 9b535738d80eb21fd42ae59f33b88ef291c8ec2e.

I plan to check test result for this.